Cabal/Distribution/Compat/ReadP.hs

```diff
@@ -297,6 +298,11 @@ skipSpaces =
   skip (c:s) | isSpace c = do _ <- get; skip s
   skip _                 = do return ()
 
+skipSpaces1 :: ReadP r ()
+-- ^ Like 'skipSpaces' but succeeds only if there is at least one
```

Comment:
BTW, why do Haddock comments in this file use this strange format? Everywhere else we use

```
-- | Foo bar baz [...]
quux :: Foo -> Bar -> [...]
```

This is not something you need to fix, I'm just wondering aloud.